tests/unit/Search/buildCardFilterDataTest.ts

```diff
@@ -0,0 +1,235 @@
+            lastFourPAN: '1138',
+        },
+        '21588684': {
+            accountID: 1,
+            bank: 'Expensify Card',
+            cardID: 21588684,
+            cardName: '',
+            domainName: 'expensify-policy2.exfy',
+            lastFourPAN: '',
+        },
+    },
+    'cards_11111_Expensify Card': {
+        '21589168': {
+            accountID: 1,
+            bank: 'Expensify Card',
+            cardID: 21589168,
+            cardName: '455594XXXXXX4163',
+            domainName: 'expensify-policy3.exfy',
+            lastFourPAN: '4163',
+        },
+        '21589182': {
+            accountID: 1,
+            bank: 'Expensify Card',
+            cardID: 21589182,
+            cardName: '',
+            domainName: 'expensify-policy3.exfy',
+            lastFourPAN: '',
+        },
+        '21589202': {
+            accountID: 1,
+            bank: 'Expensify Card',
+            cardID: 21589202,
+            cardName: '455594XXXXXX6232',
+            domainName: 'expensify-policy3.exfy',
+            lastFourPAN: '6232',
+        },
+        '21638322': {
+            accountID: 1,
+            bank: 'Expensify Card',
+            cardID: 21638322,
+            cardName: '',
+            domainName: 'expensify-policy3.exfy',
+            lastFourPAN: '',
+        },
+    },
+};
+
+const cardList = {
+    '21588678': {
+        accountID: 1,
+        bank: 'Expensify Card',
+        cardID: 21588678,
+        cardName: '455594XXXXXX1138',
+        domainName: 'expensify-policy2.exfy',
+        lastFourPAN: '1138',
+    },
+    '21588684': {
+        accountID: 1,
+        bank: 'Expensify Card',
+        cardID: 21588684,
+        cardName: '',
+        domainName: 'expensify-policy2.exfy',
+        lastFourPAN: '',
+    },
+    '21589202': {
+        accountID: 1,
+        bank: 'Expensify Card',
+        cardID: 21589202,
+        cardName: '455594XXXXXX6232',
+        domainName: 'expensify-policy3.exfy',
+        lastFourPAN: '6232',
+    },
+    '21604933': {
+        accountID: 1,
+        bank: 'vcf',
+        cardID: 21604933,
+        cardName: '480801XXXXXX1601',
+        domainName: 'expensify-policy1.exfy',
+        lastFourPAN: '1601',
+    },
+    '11111111': {
+        accountID: 1,
+        bank: 'Expensify Card',
+        cardID: 11111111,
+        cardName: '455594XXXXXX1138',
+        domainName: 'testDomain',
+        lastFourPAN: '1138',
+    },
+};
+
+const domainFeedData = {testDomain: {domainName: 'testDomain', bank: 'Expensify Card', correspondingCardIDs: ['11111111']}};
+
+function translateMock(key: string, obj: {cardFeedBankName: string; cardFeedLabel: string}) {
+    if (key === 'search.filters.card.expensify') {
+        return 'Expensify';
+    }
+    return `All ${obj.cardFeedBankName}${obj.cardFeedLabel ? ` - ${obj.cardFeedLabel}` : ''}`;
+}
+
+describe('Build individual cards data from given cardList and workspaceCardFeeds objects', () => {
```

Comment:
just do `describe('buildCardFeedsData')` and add at least one case with wrong data, for example empty cardFeeds or something